packages/repositories.config

```diff
@@ -2,9 +2,14 @@
 <repositories>
   <repository path="..\Octokit.Reactive\packages.config" />
   <repository path="..\Octokit.Tests.Conventions\packages.config" />
+  <repository path="..\Octokit.Tests\packages.Octokit.Tests.config" />
```

Comment:
@naveensrinivasan sorry, I should have been clearer here - these new entries are just duplicates of previous ones - can you :fire: them to simplify the diff?